.env.example

```diff
@@ -5,7 +5,8 @@ LOG_LEVEL=info
 ## Messaging Microservice ##
 MESSAGING_SERVICE_PORT=4001
 
-MESSAGING_SERVICE_AUTOMATED_TEST_JWT=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InVzZXIwIiwiZW1haWwiOiJ0ZXN0QHRlc3QuY29tIiwiYWRtaW4iOnRydWV9.X_SzIXZ-oqEL67eB-fwFqFSumuFQVAqhgsmak1JLIWo
+MESSAGING_SERVICE_AUTOMATED_TEST_JWT = eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NDUsInVzZXJuYW1lIjoidXNlcjAiLCJlbWFpbCI6InRlc3RAdGVzdC5jb20iLCJhZG1pbiI6dHJ1ZSwic2NvcGVzIjpbImNsaW5pY2lhbiIsImFkbWluIl0sImlhdCI6MTU0MjIwODIwMiwiZXhwIjozNjAwMDAwMDAwMDE1NDIyMDAwMDB9.e0sfocRLlARNT95R3P4rfS440t_r9K-QHQxDOlqYkB4
+MESSAGING_SERVICE_THREAD_SCOPES=["admin", "clinician"]

```

Comment:
Should I include `programAdministrator` in the list of scopes that can create and reply to threads?

Reply:
for the `.env.example` no. This can be configured in our orange deployment if we want that.